src/client/VZSidebar/Item.tsx

```diff
@@ -7,6 +7,8 @@ import {
 
 // TODO consider moving this up to a higher level of the component tree
 import { DeleteConfirmationModal } from './DeleteConfirmationModal';
+import { Trash } from '../Icons/Trash';
+import { Button } from 'react-bootstrap';
```

Comment:
```suggestion
import { Button } from '../bootstrap';
```

The react-bootstrap imports need to be done in a certain way to support Vite's build